proto/satellite.proto

```diff
@@ -264,3 +264,47 @@ message SatOpRow {
     // check nulls_bitmask to differentiate between the two
     repeated bytes values = 2;
 }
+
```

Comment:
We need to update minor version of this protocol in the proto file and regenerate satellite.ts based on that. Also potentially modify a few places where this version is referenced (mainly I expect  proto.ts)